wal_steam.py

```diff
@@ -73,15 +73,15 @@ def setCustomStyles(colors, variables, walColors, alpha, steam_dir, fonts = []):
     except FileNotFoundError:
         print("No file to remove")
 
-    f = open(METRO_COLORS_FILE, 'r')
```

Comment:
This cleanup looks great for the most part, but for the file opens, I noticed you removed the `f.close()` line. Is `f.close()` no longer needed for a `with open()`?

Reply:
That's right. In Python, the `with` keyword is how one uses a context manager. Once Python exits the `with` block, the file is automatically closed.

https://www.pythonforbeginners.com/files/with-statement-in-python

Reply:
Awesome merging. Thanks for the cleanup!